src/components/corn-chart.tsx

```diff
@@ -26,12 +26,18 @@ export default class CornChart extends React.Component<IProps, IState> {
                       planted: x.initial.countCorn,
                       harvested: x.final && x.final.countCorn
                     }));
+    // always show five years of data
+    while (cornData.length < 5) {
+      cornData.push({ year: `Year ${cornData.length + 1}`, planted: 0, harvested: 0 });
+    }
+
     return (
-    	<BarChart width={300} height={180} data={cornData} barGap={0}
+      <BarChart className="corn-chart"
+            width={width} height={height} data={cornData} barGap={0}
             margin={{top: 5, right: 30, left: 20, bottom: 5}}>
```

Comment:
Do these charts require in-line styling or can this be moved to css? Not hugely urgent for this phase of development, mostly curious about this library.

Reply:
Good question. I didn't take the time to investigate. At that point in the process I was just looking for the quickest/easiest way to get a bar chart onto the screen. 😉